### PR TITLE
Swaps live migration to limited live migration

### PR DESCRIPTION
**Pull request by stevsmit**
This is a simple name change. 

Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10662 final followup

Link to docs preview:
https://77956--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html
https://77956--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM - Only the one small nit about "+" sign....

File: modules/live-migration-metrics-information.adoc [hunk 1]
```diff
@@ -4,29 +4,29 @@
...
 The list of label values includes the following::
 +
 --
 * `UnsupportedCNI`: Unable to migrate to the unsupported target CNI. Valid CNI is `OVNKubernetes` when migrating from OpenShift SDN.
-* `UnsupportedHyperShiftCluster`: Live migration is unsupported within an HCP cluster.
-* `UnsupportedSDNNetworkIsolationMode`: OpenShift SDN is configured with an unsupported network isolation mode `Multitenant`. Migrate to a supported network isolation mode before performing live migration.
+* `UnsupportedHyperShiftCluster`: Limited live migration is unsupported within an HCP cluster.
+* `UnsupportedSDNNetworkIsolationMode`: OpenShift SDN is configured with an unsupported network isolation mode `Multitenant`. Migrate to a supported network isolation mode before performing limited live migration.
 * `UnsupportedMACVLANInterface`: Remove the egress router or any pods which contain the pod annotation `pod.network.openshift.io/assign-macvlan`. 
 Find the offending pod's namespace or pod name with the following command: +
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply modules/live-migration-metrics-information.adoc:31]: @GroceryBoyJr I have to have ti there to get the line break below :D

**Pull request merged**